src/main/java/VOConfigTask.java

```diff
@@ -551,12 +514,11 @@ public void endElement(String uri, String localName, String qName) throws SAXExc
                             vomsServers.put(VOMSServerKey,vomsServer);
                         }
                         vomsEndpoint.setServer(vomsServers.get(VOMSServerKey));
-                        //vomsEndpoint.setEndpoint("vomss://" + vomsServer.getHost() + ":" + Integer.toString(vomsServer.getPort()) + "/voms/" + voConfig.getName() + "?/" + voConfig.getName());
+                        //vomsEndpoint.setEndpoint("vomss://" + vomsServer.getHost()bae + ":" + Integer.toString(vomsServer.getPort()) + "/voms/" + voConfig.getName() + "?/" + voConfig.getName());
```

Comment:
Ah yes I'll remove that. With my current setup, I have issues with cursor moving without my intent while typing.